OTKit/otkit-colors/token.yml

```diff
@@ -1,21 +1,38 @@
 props:
+  # Primary Action
+  # This color is only used on objects, backgrounds with large text (24px or larger, bold), and will be used as a color to guide users towards actionable content.
+  # =============================================
   color-primary:
-    value: "{!color-primary}"
+    value: "#DA3743"
   color-primary-active:
-    value: "{!color-primary-active}"
+    value: "#FB665F"
   color-primary-disabled:
-    value: "{!color-primary-disabled}"
+    value: "rgba(218, 55, 67, 0.40)"
+
+  # Primary Gray, used on all text sizes
+  # =============================================  
   color-gray-primary:
-    value: "{!color-gray-primary}"
+    value: "#333333"
+
+  # Secondary Gray, used on smaller font sizes
+  # =============================================
   color-gray-secondary:
-    value: "{!color-gray-secondary}"
+    value: "#717171"
+
+  # Utility Gray, used for borders, dividers, containers
+  # =============================================
   color-gray-utility:
-    value: "{!color-gray-utility}"
-  color-white:
-    value: "{!color-white}"
+    value: "#E1E1E1"
+
+  # Lightest gray, used for backgrounds for the page and backgrounds for active/hover states
+  # =============================================
+  color-gray-active:
```

Comment:
Thanks for adding this. This is another problem with excessive aliasing that you discovered: humans are not perfect and we sometimes forget to map a variable, and the system doesn't remind us for that. 👍